File: python/src/estruturas/repetitivas/sequencia_impares.py
""" Problema "sequencia_impares"
Leia um valor inteiro X.
Em seguida mostre os ímpares de 1 até X, um valor por linha, inclusive o X, se for o caso.
"""


def sequencia_impares() -> None:
    x = int(input("Digite o valor de X (maior que 1): "))
    if x <= 1:
        x = 2

    for i in range(1, x + 1):
        if i % 2 == 1:
            print(f"{i}")


if __name__ == "__main__":
    sequencia_impares()

File: java/src/estruturas/repetitivas/SequenciaImpares.java
package estruturas.repetidas;

import java.util.Scanner;

/**
 * Problema "sequencia_impares"
 * Leia um valor inteiro X.
 * Em seguida mostre os ímpares de 1 até X, um valor por linha, inclusive o X, se for o caso.
 */
public class SequenciaImpares {
  /**
   * The entry point of application.
   *
   * @param args the input arguments
   */
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);

    System.out.print("Digite o valor de X (maior que 1): ");

    int x = sc.nextInt();
    x = x <= 1 ? 2 : x;

    for (int i = 1; i <= x; i++) {
      if (i % 2 == 1) {
        System.out.printf("%d%n", i);
      }
    }

    sc.close();
  }
}